sources.cfg

```diff
@@ -2,9 +2,7 @@
 extends = http://plonesource.org/sources.cfg
 
 auto-checkout =
-    ftw.publisher.core
-
-# Wait for first release
+# Wait for release of https://github.com/4teamwork/ftw.simplelayout/commit/5533a3a2cb9fee8748308e294d1735d38f11d026
```

Comment:
is someone going to cut a release in the near future? If so I'd wait and update this PR.

Reply:
Thanks! I'll check with @maethu regarding the release.

Reply:
Go for the release 😄

Reply:
👍 if Pypi's caching is too slow test might fail just now. Otherwise I think we are ready now.

Reply:
I approved already ;-)